src/app/build.gradle

```diff
@@ -6,18 +6,6 @@ apply plugin: 'application'
 
 mainClassName = "org.imperial.mrc.hint.AppStartKt"
 
-jacoco {
```

Comment:
Noticed that this is already declared in the parent project file for all subprojects, so didn't need to be here.